src/styles/sidebar/_base.scss

```diff
@@ -32,6 +32,6 @@
 
   // TODO: Research history of this and whether it's still appropriate
   p {
-    hyphens: auto;
+    @apply hyphens-auto;
```

Comment:
What is the benefit of using a Tailwind class here vs a CSS rule? Future proofing against adding additional CSS rules to this utility?

Reply:
I needed a Tailwind hyphen rule for migrating `GroupMenuItem` (or else I'd have to use external CSS for that component). Since I was creating one for that component, I thought it'd be more consistent to also use one here. It's arguable, maybe I'm just being overzealous :).